src/06-positron-rag/lib/insert/prepareArticlesCollection.ts

```diff
@@ -120,7 +130,7 @@ export async function prepareArticlesCollection() {
         },
       },
       {
-        name: "body",
+        name: "articleDescription",
```

Comment:
Previously we were only summarizing the article summary and calling that the `body` — I'm renaming that to something more descriptive here.